cloudformation/find-workers.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh -e
+
+if [ -z "$AWS_DEFAULT_REGION" ]; then
+    export AWS_DEFAULT_REGION=$(curl -s http://169.254.169.254/latest/dynamic/instance-identity/document | grep region | awk -F\" '{print $4}')
```

Comment:
Yep, this is a link-local IP address that the EC2 host machines use for making metadata available to EC2 instances. It will never change.